src/Pages/ChangePassword.cshtml.cs

```diff
@@ -43,14 +45,16 @@ public class ChangePasswordModel : PageModel
         private EmailSender EmailSender { get; }
         private CurrencyConverter CurrencyConverter { get; }
         public UserModel UserModel { get; private set; }
+        private IMemoryCache Cache { get; }
 
-        public ChangePasswordModel(PiggyDbContext piggyDbContext, IOptions<EmailSender> emailSenderSettings, ILogger<ChangePasswordModel> logger, CurrencyConverter currencyConverter)
+        public ChangePasswordModel(PiggyDbContext piggyDbContext, IOptions<EmailSender> emailSenderSettings, ILogger<ChangePasswordModel> logger, CurrencyConverter currencyConverter, IMemoryCache cache)
```

Comment:
This line does not fit the screen on github, so I think it would be good to chop it.